runtime/centrifuge/src/xcm.rs

```diff
@@ -309,11 +311,32 @@ pub type XcmRouter = (
 	XcmpQueue,
 );
 
+const MOONBEAM_PARA_ID: u32 = 2004;
+/// https://chainlist.org/chain/1284
+const MOONBEAM_EVM_ID: u64 = 1284;
+
+/// A constant way of mapping parachain IDs to EVM-chain IDs
+pub struct ParaToEvm;
+impl TryConvert<cfg_types::ParaId, EVMChainId> for ParaToEvm {
+	type Error = cfg_types::ParaId;
+
+	fn try_convert(a: cfg_types::ParaId) -> Result<EVMChainId, cfg_types::ParaId> {
+		// NOTE: Currently only supported moonbeam
+		match a {
+			MOONBEAM_PARA_ID => Ok(MOONBEAM_EVM_ID),
```

Comment:
Will this always match ? I am sometimes not sure how match works with integers.

Reply:
It will! See [here](https://doc.rust-lang.org/reference/expressions/match-expr.html).

Reply:
So it is correct? ^^ I want it to ONLY match when it is a MOONBEAM_PARA_ID.

Reply:
> So it is correct? ^^ I want it to ONLY match when it is a MOONBEAM_PARA_ID.

Yes, this is exactly what is happening right now. If you want to be over 9000% sure, a simple test would suffice.